scripts/setupdeps.sh

```diff
@@ -7,6 +7,11 @@ SUPPORTED_OS=('win64' 'maci64' 'maca64' 'glnxa64')
 DISTDIR="$(pwd)/dist/bin"
 mkdir -p $DISTDIR
 
+# Create plugins directory and copy buildtool plugins
+PLUGINDIR="$(pwd)/.matlab/plugins/+matlab/+ciplugins/+github"
+mkdir -p $PLUGINDIR
+cp -R plugins/+matlab/+ciplugins/+github/*.m $PLUGINDIR/
+
```

Comment:
It feels potentially unnecessary for us to have two directories that we're copying data into that don't have the same root. Maybe we can put the plugins in `DISTDIR` defined above, or update where we download `run-matlab-command` to be inside of the `.matlab` temp directory that we're creating for the plugins.

As a side note, are we cleaning up the `DISTDIR` before the end of the step right now? Or are we just leaving it?